docassemble/ALToolbox/al_income.py

```diff
@@ -338,24 +403,30 @@ def net_total(self, times_per_year=1, source=None):
 
 class ALAsset(ALIncome):
     """
-    An ALAsset represents an asset that has a market value and a balance. Can be
-        stored in an ALAssetList.
-
-    Attributes:
-    .market_value {num} Market value of an asset.
-    .balance {num} Balance of an asset.
-    .times_per_year {str | num} Represents a number of the annual frequency of
-        the value. E.g. 12 for a monthly value.
-    .value {num} (Optional) Represents an amount of money accumulated by this
-        asset over its `times_per_year`. Default is 0.
+    An ALAsset represents an asset that has a market value, an optional income
+    that the asset earns, and an optional balance which may be helpful if the
+    asset represents a financial account rather than a physical asset.
+
+    Can be stored in an ALAssetList.
+
+    Attributes: .market_value {float | Decimal} Market value of the asset.
```

Comment:
Is this missing a new line character?
```suggestion
    Attributes:
     .market_value {float | Decimal} Market value of the asset.
```